### PR TITLE
main

### LINKED ISSUE #22050: batch_norm

### PR DESCRIPTION
**Pull request by Ambro19**
Close #22050

I uncommitted the changes I have made inside the array_api_testing folder. Also, I removed all the unrelated changes from this PR.

### CODE REVIEW

File: ivy/functional/frontends/paddle/nn/functional/norm.py [hunk 1]
```diff
@@ -8,3 +8,71 @@
...
+    running_var=None,
+    weight=None,
+    bias=None,
+    use_input_stats=True,
+    momentum=0.9,
+    epsilon=1e-05,
+    data_format="NCHW",
+    name=None,
+):
+    r"""
```
Comment (DragosStoican) [inline ivy/functional/frontends/paddle/nn/functional/norm.py:28]: There is no need for docstrings in the frontend API, can you please remove them. Thanks!

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py [hunk 2]
```diff
@@ -43,3 +43,36 @@ def test_paddle_layer_norm(
         bias=offset[0],
         epsilon=eps,
     )
+
+
+# instance_norm
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.instance_norm",
+    values_tuple=_generate_data_layer_norm(
+        available_dtypes=helpers.get_dtypes("float"),
```
Comment (DragosStoican) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py:52]: We should test for all "valid" data types, by using the keywork 'valid' here instead of 'float'

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py [hunk 3]
```diff
@@ -43,3 +43,36 @@ def test_paddle_layer_norm(
...
+):
+    (dtype, x, normalized_shape, scale, offset) = values_tuple
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        on_device=on_device,
+        fn_tree=fn_tree,
+        x=x[0],
+        normalized_shape=normalized_shape,
```
Comment (DragosStoican) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py:74]: We should be testing for running_mean, running_var, use_input_stats, momentum and data format. All parameters of `paddle.nn.functional.instance_norm` must be included here. Also, there is no `normalized_shape` in `instance_norm`.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.
